frontend/talentsearch/src/js/components/Browse/BrowsePoolsPage.tsx

```diff
@@ -0,0 +1,327 @@
+import React from "react";
+import { RocketLaunchIcon } from "@heroicons/react/24/outline";
+import { useIntl } from "react-intl";
+
+import { CardFlat } from "@common/components/Card";
+import Flourish from "@common/components/Flourish";
+import Hero from "@common/components/Hero";
+import Heading from "@common/components/Heading";
+import Link from "@common/components/Link";
+import Pending from "@common/components/Pending";
+import NotFound from "@common/components/NotFound";
+
+import { commonMessages } from "@common/messages";
+import { imageUrl } from "@common/helpers/router";
+import useTheme from "@common/hooks/useTheme";
+
+import PoolCard from "./PoolCard";
+
+import {
+  AdvertisementStatus,
+  PublishingGroup,
+  PoolAdvertisement,
+  useBrowsePoolAdvertisementsQuery,
+} from "../../api/generated";
+import TALENTSEARCH_APP_DIR from "../../talentSearchConstants";
+import useBreadcrumbs from "../../hooks/useBreadcrumbs";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+import { useApplicantProfileRoutes } from "../../applicantProfileRoutes";
+
+const flourishTopLight = imageUrl(TALENTSEARCH_APP_DIR, "browse_top_light.png");
+const flourishBottomLight = imageUrl(
+  TALENTSEARCH_APP_DIR,
+  "browse_bottom_light.png",
+);
+const flourishTopDark = imageUrl(TALENTSEARCH_APP_DIR, "browse_top_dark.png");
+const flourishBottomDark = imageUrl(
+  TALENTSEARCH_APP_DIR,
+  "browse_bottom_dark.png",
+);
+
+const getFlourishStyles = (isTop: boolean) => ({
+  "data-h2-position": "base(absolute)",
+  "data-h2-width": "base(25vw)",
+  "data-h2-offset": isTop ? "base(0, 0, auto, auto)" : "base(auto, auto, 0, 0)",
+});
+
+export interface BrowsePoolsProps {
+  poolAdvertisements: PoolAdvertisement[];
+}
+
+export const BrowsePools: React.FC<BrowsePoolsProps> = ({
+  poolAdvertisements,
+}) => {
+  const { mode } = useTheme();
```

Comment:
It looks really nice but is this sufficient contrast to be accessible?
![image](https://user-images.githubusercontent.com/8978655/196707122-e151b2af-a911-4255-9a2f-65a958dab373.png)
Probably not a blocker for this PR since we haven't deployed the theme switcher yet but we'll need to review these pages before we do.

Reply:
So this was working when I passed it - @esizer did the font color value on the parent wrapper for the page get moved elsewhere?

Reply:
Yeah, looks like I lost it. 

Updated in [update font color in dark mode](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4293/commits/19c2e94c9285f7a2ddaef5113ea9c8f5d2a19d5d)